src/main/java/view/panels/OrderEntryPanel.java

```diff
@@ -128,6 +129,16 @@ private void handleBuyAction() {
         quantityInput.clear();
     }
 
+    private void handleSellAction() {
+        String ticker = tickerInput.getText();
+        String quantity = quantityInput.getText();
+
+        // Execute sell action
+        ExecuteSellController controller = ServiceManager.Instance()
+                .getService(ExecuteSellController.class);
+        controller.execute(ticker, quantity);
+    }
```

Comment:
This input fields should be cleared, check handleBuyAction.

Reply:
Consider providing user feedback after the sell operation is executed, such as a confirmation message or an error notification, to ensure the user is informed about the outcome of their action.